src/app/shared/services/search/search.service.ts

```diff
@@ -24,7 +24,7 @@ import { SearchState } from '../../state/search/search.state';
  */
 function createSearchCriteriaDescription({ ageRange: [minAge, maxAge], gender }: SearchStateModel): string {
   let ageDescription = `Age ${defaultTo(minAge, 18)}-${defaultTo(maxAge, '89+')}`;
-  if (minAge === maxAge || minAge === 89 || maxAge === 18) {
+  if (minAge === maxAge || minAge === 90 || maxAge === 18) {
     if (minAge) {
       ageDescription = `Age ${minAge}`;
```

Comment:
This should show 89+

Reply:
if (minAge === maxAge || minAge >= 89 || maxAge === 18) {

is this alright?

Reply:
Yes, but ageDescription needs to be modified as well

Reply:
ok, i will fix this and also externalize the magic numbers.

Reply:
Thanks